### ISSUE TITLE
Can you give me a simple example   Vue with Golang ?

### ISSUE THREAD

**Issue by AttSee** [thewhitetulip/Tasks-vue#1]
Thanks great work
 Can you give me a simple example   Vue with Golang ?

**Comment by thewhitetulip** [comment]
Read this and let me know if this helps, https://github.com/thewhitetulip/intro-to-vuejs/

**Comment by AttSee** [comment]
Sorry @thewhitetulip  for the late response. 

Golang uses {{}} 
vuejs uses {{}}
I change delimiters
new Vue({
delimiters: ['${', '}']
})
yes this work 
but not working with golang
I'm trying to understand why not work for me 
I need a very simple(Hello world) example this will help me thank you

**Comment by thewhitetulip** [comment]
Yes, it seems you have not read the book which this app is based out of, read the link I gave in previous comment

**Comment by AttSee** [comment]
I will :)

**Comment by thewhitetulip** [comment]
Please do :) This is the reason why I wrote that book!
